android/source/VideoLocker/build.gradle

```diff
@@ -232,7 +232,7 @@ android {
 
             versionCode 40
             versionName '1.0.03'
-            minSdkVersion 16
+            minSdkVersion 14
             targetSdkVersion 21
```

Comment:
Ok, we will make these changes when configuring the app for automated build generation. Looks good for now!